src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
+        String randomColor = Color.values()[index].name();
+        return randomColor.toLowerCase();
```

Comment:
```suggestion
    private final Random random = new Random();

    public String getRandomColor() {
        int index = random.nextInt(Color.values().length);
        String randomColor = Color.values()[index].name();
        return randomColor.toLowerCase();
```
Random should be a class variable